functests-performance-profile-creator/1_performance-profile_creator/ppc.go

```diff
@@ -109,6 +110,81 @@ var _ = Describe("[rfe_id:OCP-38968][ppc] Performance Profile Creator", func() {
 		}
 	})
 
+	It("[test_id:41419] Verify PPC script fails when reserved cpu count is 2 and requires to split across numa nodes", func() {
```

Comment:
We need to add context here: On a system with hyperthreading enabled...

Reply:
I have added the context now.